Source/ARTRealtimeChannel.m

```diff
@@ -69,6 +69,10 @@ - (ARTRealtimeChannelState)state {
     return _internal.state;
 }
 
+- (ARTChannelProperties *)properties {
```

Comment:
I think it would be good to have a test of this property.

Reply:
https://github.com/ably/ably-cocoa/pull/1965/commits/80180c3550fe000fc8aac22b1a42fdb5d9212a53

Reply:
That doesn't test the spec point; it just tests that the public properties match some internal properties, no?

Reply:
Yes, but tests testing `attachSerial` and `channelSerial` while attaching are updated to use `properties`. Isn't it what you were asking?

Reply:
I’m not sure what you mean by that. What I was asking for was tests of spec points RTL15a and RTL15b.

Reply:
The test for RTL15b already [exists](https://github.com/ably/ably-cocoa/blob/fbe2e929597694aef0fff853d6f16447e151d15a/Test/Tests/RealtimeClientChannelTests.swift#L4241) and I've updated it. I've added RTL15a check in fee77037e6b1adff227e2fd63ffd534ea97dc7d5